utils/create-icons-map.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const path = require('path');
+
+const svgFolder = '../assets/icons/';
+const iconMapFile = '../src/iconsMap.js';
+
+let iconsObject = '';
+
+let iconMapDir = path.dirname(iconMapFile);
+if (!fs.existsSync(iconMapDir)) {
+    fs.mkdirSync(iconMapDir);
+};
+
+
+
+fs.readdirSync(svgFolder).forEach(file => {
+    let fileName = file.split('.')[0];
+    let filePath = svgFolder + file;
+    if (fs.existsSync(filePath)) {
+        const iconBody = fs.readFileSync(filePath).toString();
+        const cleanedIcon = preparIcon(iconBody, fileName);
+        iconsObject += "'" + fileName + '\' : \'' + cleanedIcon + '\', \n';
+    }
+})
+
+const dataToWrite = 'export const Icons = {' + iconsObject + '};';
+
+fs.writeFileSync(iconMapFile, dataToWrite);
```

Comment:
I don't see it safe to output a JS file and then import it.

Reply:
we made it because we want to use it also for ReactApp

Reply:
So why not JSON? it's much safe.

Reply:
It was recommendation of Israel. I could make it like JSON. @ArielDKenan What you think about it?

Reply:
Only make sure to parse it once, and not for every single icon.

Reply:
We can consume a JSON file, as well. Either way is fine. I like your idea of exporting a JSON for either app to consume